src/test/txpackage_tests.cpp

```diff
@@ -658,7 +662,7 @@ BOOST_FIXTURE_TEST_CASE(package_cpfp_tests, TestChain100Setup)
     // Package feerate is calculated using modified fees, and prioritisetransaction accepts negative
     // fee deltas. This should be taken into account. De-prioritise the parent transaction by -1BTC,
```

Comment:
d32f445ab9f9a65d7bd0eec828b8a3de0d07d635
the `-1BTC comment` is irrelevant now?

Reply:
Amended to remove the `-1BTC`